js/main.js

```diff
@@ -0,0 +1,345 @@
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+},
+];
+
+const projectList = document.querySelector('#portfolio ul.grid');
+
+function loadProjects() {
+  for (let i = 0; i < projects.length; i += 1) {
+    const projectItemtemplate = `
+    <li class="project-item ${projects[i].class}">
+      <img
+        class="project-img"
+        src="${projects[i].image}"
+        alt="laptop"
+      />
+      <div class="container container-information hover">
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
+          </div>
+          <ul role="list" class="modal__header_bottom features-container flex">
+            <li class="feature">${projects[id].features[0]}</li>
+            <li class="feature">${projects[id].features[1]}</li>
+            <li class="feature">${projects[id].features[2]}</li>
+          </ul>
+        </div>
+        <div class="modal__body">
+          <div class="modal__body_slide">
+            <div class="slide_main">
+              <span class="slide_prev flex">&lt;</span>
+              <div class="slide_show">
+                <img src="${projects[id].image}" />
+              </div>
+              <span class="slide_next flex">&gt;</span>
+            </div>
+            <ul class="slide_items flex" role="list">
+              <li class="slide_item active">
+                <img src="${projects[id].sideImage[0]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[1]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[2]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[3]}" alt="project" />
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] Good job adding the project in the modal window 🥇, but please remove the white space that is below, it is not present in the original design. 🔍

![image](https://user-images.githubusercontent.com/91301423/202871407-30738de9-0b65-4dc5-add0-30deac3f14f1.png)